packages/wallets/talisman/src/helpers.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  Chain,
+  ChainToHexChainId,
+  addEVMWalletNetwork,
+  prepareNetworkSwitch,
+} from "@swapkit/helpers";
+
+import {
+  ARBToolbox,
+  BSCToolbox,
+  BrowserProvider,
+  ETHToolbox,
+  type Eip1193Provider,
+  MATICToolbox,
+  OPToolbox,
+} from "@swapkit/toolbox-evm";
+
+import { decodeAddress, encodeAddress } from "@polkadot/util-crypto";
+
+import { getToolboxByChain } from "@swapkit/toolbox-substrate";
+import type { InjectedWindow } from "./types";
+
+// Function to convert an address to a different format
```

Comment:
It's self explanatory comment of what code does - unnecessary
```suggestion
```

Reply:
fixed